state/state.go

```diff
@@ -58,4 +58,8 @@ func (a Address) MarshalJSON() ([]byte, error) {
 }
 
 // PayForBlobOptions contains additional options that will be applied to the `MsgPayForBlob`.
-type PayForBlobOptions struct{}
+type PayForBlobOptions struct {
+	// Specifies the key that should be used to sign transactions.
+	// NOTE: This `AccName` should be available in the `Keystore`.
+	AccName string
```

Comment:
I would call it `KeyName` so it's more clear what we are actually looking for.

Reply:
speaking of which, we should eventually change the name of the `--keyring.accname flag` to `--keyring.keyname`

Reply:
Should we break it in this release? @liamsi @Wondertan 

I feel like `--keyring.keyname` is more clear and it'd prevent us having to break in future

Reply:
also would change the name of KeyringAccName in state config (which is breaking as well)

we shouldn't have used this terminology from sdk / comet to begin with.